### PR TITLE
Fix broken functions in ArticulationView

### PR DESCRIPTION
**Pull request by cgokmen**
This copies 3 more functions verbatim from Isaac's ArticulationView and fixes their implementations. See my issue on NVIDIA developer forums for more details. https://forums.developer.nvidia.com/t/articulationview-bugs-in-joint-position-velocity-effort-setting/279186

### CODE REVIEW

File: omnigibson/robots/manipulation_robot.py [hunk 1]
```diff
@@ -916,6 +916,7 @@ def _freeze_gripper(self, arm="default"):
         for joint_name, j_val in self._ag_freeze_joint_pos[arm].items():
             joint = self._joints[joint_name]
             joint.set_pos(pos=j_val)
+            joint.set_vel(vel=0.0)
```
Comment (cremebrule) [inline omnigibson/robots/manipulation_robot.py:919]: why is this needed?

**Review by cremebrule — APPROVED** [review]
lgtm, just one comment! feel free to merge afterwards

File: omnigibson/robots/manipulation_robot.py (hunk 1 above)
Comment (cgokmen) [reply omnigibson/robots/manipulation_robot.py:919]: This used to be here but was removed recently by Wensi attempting to fix this issue. This solves it so we can put it back. It stops the gripper from moving out.
